checker/jdk/nullness/src/java/util/Vector.java

```diff
@@ -880,7 +881,7 @@ public void clear() {
      * @throws NullPointerException if the specified collection is null
      */
     @Pure
-    public synchronized boolean containsAll(Collection<?> c) {
+    public synchronized boolean containsAll(Collection<? extends @NonNull Object> c) {
```

Comment:
The docs suggest that these `Vector` bulk operations throw NPE only if the _collection_ is `null`. They do look like they accept null _element_.